pkg/config/templates/config.toml

```diff
@@ -304,6 +304,15 @@ enabled = false
 # Obtain from https://clerk.com/setup/supabase
 # domain = "example.clerk.accounts.dev"
 
+# OAuth server configuration
+[auth.oauth_server]
```

Comment:
Let's also add this block to [testdata](https://github.com/supabase/cli/blob/develop/pkg/config/testdata/config.toml) with `enabled = true` so it's covered by tests.